### PR TITLE
Minor cleanup and styling

### PR DESCRIPTION
**Pull request by Tourorist**
1. Rasterized and lined up the Shields logo to leading badges and subtitle;
2. Alphabetized the user list;
3. Verified the official project names:
 * Scrutinizer, minus CI
 * Version Badge vs Gemfury/RubyGems
 * Badger vs BadgerBadgerBadger
 * SemVer vs semver
 * Gratipay vs gratipay
4. Bunched the documentation links into a list.

### CODE REVIEW

**Comment by Tourorist** [comment]
* [Preview](https://github.com/Tourorist/shields/blob/master/README.md) - for an idea of how this version looks, since the logo ([serparate PR](https://github.com/badges/shields/pull/484)) is missing here.

File: README.md [hunk 1]
```diff
@@ -45,39 +55,35 @@ As you can see below, without increasing the footprint of these badges, I've tri
...
+* [Read the Docs](https://readthedocs.org/)
+* [reposs][]
+* [ruby-gem-downloads-badge][]
+* [Scrutinizer](https://scrutinizer-ci.com/)
+* [Semaphore](https://semaphoreapp.com)
+* [Travis CI](https://github.com/travis-ci/travis-ci/issues/630#issuecomment-38054967)
+* [Version Badge](http://badge.fury.io/)
+* [VersionEye]
 
 [gem]: https://github.com/badges/badgerbadgerbadger
```
Comment (d10) [inline README.md]: The format for URL linking is a little inconsistent. Perhaps you should place them all inline instead of at the bottom?

**Comment by espadrine** [comment]
That looks great!

Could you merge the patches  in https://github.com/badges/shields/pull/484 and here, to have a single PR?

Did you try to have the Shields logo as SVG? DId it fail to work in the readme file?

File: README.md (hunk 1 above)
Comment (Tourorist) [inline README.md]: @d10, I'd like to as well, just didn't know if there was a special reason for stacking those select at the bottom. @espadrine, can I bring them inline, or is it meant to be the way it is now?

**Comment by Tourorist** [comment]
SVG by itself works but scaling it is beyond my feeble mind. :smiley:

Just glancing over [some tutorials](https://css-tricks.com/scale-svg/), it appears to be done through the &lt;svg&gt; element in combination with `viewBox` attribute. Don't know if that works with MD, but if it is possible then there wouldn't be any need for that other PR. I'll see if it's doable.

**Comment by Tourorist** [comment]
Combined the list highlighting issues with existing badges into a table, saving on some vertical space (less scrolling). On a subjective plus side, visually, it partitions its content better than a plain list.

**Comment by Tourorist** [comment]
Was kindly informed by @cmrberry that due to security concerns SVG rendering is currently a blacklisted feature. So yeah, with the .png version now in tow of this PR, I'd say that it is ready to be merged.

But before you do that, @espadrine, do you want me to patch up the issue raised by @d10, or anything else while I am at it?

**Comment by espadrine** [comment]
ugghhh I feel so bad, two weeks! I guess work was intense, but it's a really lame excuse, and not a very valid one at that.

I think @d10's idea is pretty good. I personally prefer the `[link][]` form.

Also, `logo/` is a directory to store the vendor logos, like Twitter's or GitHub's or… well, they're for badges. Since I compute a list of whatever is in the directory, it might have weird effects. Could you put the png as `logo.png` in the toplevel directory, just like it is currently `logo.svg`?

**Comment by Tourorist** [comment]
No problem,
Made the requested corrections.

**Comment by espadrine** [comment]
Could you remove `logo/rasterizer.png` in the same commit please?

(Oh my, such a speedy response.)

**Comment by Tourorist** [comment]
Sorry, forgot it was still there. Done.

**Comment by espadrine** [comment]
I merged it by fixing a few things.

It seems camo has a small bug when loading fonts, but I think it can be fixed; I fixed a similar one ([here's a video of it](https://www.youtube.com/watch?v=AfImO5lDlis)).

**Comment by espadrine** [comment]
Here's a link to the fix: https://github.com/atmos/camo/pull/87

**Comment by Tourorist** [comment]
Was unaware things like camo existed; it sure made my workaround completely redundant.

Thanks for sharing the smarts. :thumbsup:

**Comment by espadrine** [comment]
Heh, it's pretty cool most of the time!

**Pull request closed without merging**